integ/lib/render-struct.ts

```diff
@@ -30,6 +33,12 @@ export class RenderStruct extends Construct {
     // Retrieve VPC created for _infrastructure stack
     const vpc = Vpc.fromLookup(this, 'Vpc', { tags: { StackName: infrastructureStackName }}) as Vpc;
 
+    // Retrieve Docker image overrides, if available
+    let dockerImageOverrides: (ThinkboxDockerImageOverrides | undefined) = undefined;
+    if (process.env[DOCKER_IMAGE_OVERRIDES_ENV_VAR] !== undefined) {
+      dockerImageOverrides = ThinkboxDockerImageOverrides.fromJSON(this, 'ThinkboxDockerImageOverrides', process.env[DOCKER_IMAGE_OVERRIDES_ENV_VAR]!.toString());
+    }
+
     // Stage docker recipes, which include image used for the render queue instance
     const recipes = new ThinkboxDockerRecipes(this, 'DockerRecipes', {
```

Comment:
If the staging folder has other dependencies then I think its fine to leave it like this for now. We can handle it later. 
IMO, instead of using the docker image, we can use aws cli to copy the installer from S3 bucket directly.